os/fs/smartfs/smartfs_utils.c

```diff
@@ -1210,8 +1214,12 @@ int smartfs_find_availableentry(struct smartfs_mountpt_s *fs, uint16_t *parentdi
 			if (IS_AVAILABLE_ENTRY(entry)) {
 				direntry->dsector = psector;
 				direntry->doffset = offset;
+#ifdef CONFIG_SMARTFS_ALIGNED_ACCESS
```

Comment:
Actually I added this when I send mail to you but...we will reset flag, so this is not necessary right?

Reply:
Yes, shouldn't be necessary.